modules/claims_api/app/controllers/concerns/claims_api/v2/disability_compensation_validation.rb

```diff
@@ -400,6 +413,17 @@ def validate_form_526_separation_pay_received_date!
                                                           separation_pay_received_date)
       end
 
+      def validate_from_526_separation_severance_pay_branch!
+        branch = form_attributes.dig('servicePay', 'separationSeverancePay', 'branchOfService')
+        return if branch.nil? || brd_service_branch_names.include?(branch)
```

Comment:
couple things here.
1. this fails as it's case-sensitive...
```
(byebug) branch
"Naval academy"
(byebug) brd_service_branch_names.include?(branch)
false
(byebug) brd_service_branch_names.include?(branch.titleize)
true
```

2. How do our external services require us to send this? do we need the same approach for them or will they accept any sensitivity? This is more of a food for thought.